packages/openapi-ts/test/__snapshots__/test/generated/v3_date/schemas.ts.snap

```diff
@@ -1,45 +1,14 @@
 export const $ModelWithPattern = {
     description: `This is a model that contains a some patterns`,
     properties: {
-        key: {
-            type: 'string',
-            isRequired: true,
-            maxLength: 64,
-            pattern: '^[a-zA-Z0-9_]*$',
-        },
-        name: {
-            type: 'string',
-            isRequired: true,
-            maxLength: 255,
-        },
-        enabled: {
-            type: 'boolean',
-            isReadOnly: true,
-        },
-        modified: {
-            type: 'string',
-            isReadOnly: true,
-            format: 'date-time',
-        },
-        id: {
-            type: 'string',
-            pattern: '^\\d{2}-\\d{3}-\\d{4}$',
-        },
-        text: {
-            type: 'string',
-            pattern: '^\\w+$',
-        },
-        patternWithSingleQuotes: {
-            type: 'string',
-            pattern: "^[a-zA-Z0-9']*$",
-        },
-        patternWithNewline: {
-            type: 'string',
-            pattern: 'aaa\nbbb',
-        },
-        patternWithBacktick: {
-            type: 'string',
-            pattern: 'aaa`bbb',
-        },
+        key: { type: 'string', isRequired: true, maxLength: 64, pattern: '^[a-zA-Z0-9_]*$' },
+        name: { type: 'string', isRequired: true, maxLength: 255 },
+        enabled: { type: 'boolean', isReadOnly: true },
+        modified: { type: 'string', isReadOnly: true, format: 'date-time' },
+        id: { type: 'string', pattern: '^\\d{2}-\\d{3}-\\d{4}$' },
```

Comment:
Can we figure out new lining these so the diff is easier to see. I will handle moving some of the functionality to utility functions later